src/rendering/dashboard.cpp

```diff
@@ -137,8 +149,18 @@ void Dashboard::render(glm::vec2& penPosition) {
         return;
     }
 
+    auto now = std::chrono::high_resolution_clock::now();
```

Comment:
IMHO, it should be a `steady_clock` to ensure it is not going backwards in time.